demo/server/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG SERVER_TYPE=-all-ai
+FROM deephaven/server$SERVER_TYPE:local-build
+
+# Add the extra libraries to the classpath
+COPY libs/ /app/libs
```

Comment:
Are we modifying the start script? And the location of the jars? For main atm, we have `/opt/deephaven/...`.